### PR TITLE
WI global settings style changes

### PR DESCRIPTION
**Pull request by Wolfsblvt**
Seems like quite a few people miss that the setting in the WI panel can actually be opened to show the global settings.
Suggested change to make it more obvious and rename it.
Suggestion taken from [here](https://discord.com/channels/1100685673633153084/1100685673633153087/1238269314981822467).

Looks like this now:

image
image

### CODE REVIEW

**Comment by Cohee1207** [comment]
"Global Activation Settings" and we have a deal.

**Comment by Wolfsblvt** [comment]
If we never add any general/global WI settings there, only about activation, then yes. Right now they all fit. Only the sorting insertion strategy is kinda borderline not about "activation" in itself, but eh.

Why am I arguing semantics. The UI change is the important part so you can see more easily that this is a collapsed settings category.

Now I could even see naming the second one "World Configuration" instead of "World Settings". Opinion?

**Comment by Cohee1207** [comment]
The second header could be removed at all. It does not convey any useful information.

**Comment by Wolfsblvt** [comment]
Well, it's the second header that now actually makes it look that the first one isn't just the header for the main content box below, but a **separate** header with its own content, just collapsed.

That's the whole issue, with people not noticing that this can be expanded.

**Comment by Cohee1207** [comment]
This has a simple yet elegant solution `class="standoutHeader"`
image

**Comment by Wolfsblvt** [comment]
Oh, the consistency. My heart...

Yes, I'd agree, this would work. But now look how the headers are in the one other place where we also have a collapsed-by-default header.
All the other headers are that standout, but that one isn't?
image

Then it's much closer to the Advanced panel of the character editor. Which has "normal" headings like here, but there it works because the already expanded section has its own header.
image

I don't know. There is no nice solution. But same as I didn't (and still don't) like the collapsed header int he settings page, I don't think this here in WI is a good design, if the section below does not have it's own header.
You don't really see what you collapse and expand.

I would call this nitpicking, but there were multiple people who didn't seem to get that this is a section you can actually open. That's why this PR exists :D

**Comment by Cohee1207** [comment]
Counter-argument: The extension panel has all collapsed drawers have this style.

**Comment by Cohee1207** [comment]
So the arrow should give the idea that this is clickable. Also a cursor.

**Comment by Wolfsblvt** [comment]
Yes, but they each are visibly their own section, and the section below is visibly a new section not being part of the one above.

I mean, it can work. Better than what we have right now for sure. Would work even better if we had two rows of expandable settings like in the extensions panel, then it's even more obvious, so a good chance to add more WI settings, heh.

**Review by Cohee1207 — CHANGES_REQUESTED** [review]
TL;DR
1. Global Activation Settings moniker: 👍 
2. Visible drawer header: 👍 
3. Second header for the editor: 👎

**Comment by LenAnderson** [comment]
Like wolf said, this is the same issue that exists with the user settings. Your throwing a drawer somewhere in the middle of non-drawer content and the visual distinction is not at all obvious. This is made worse by ST drawers having no separation at all between their content and other stuff around them. Drawer content just kinda exists below the header when expanded with no separation from other content.
While this is fine in the extension settings where everything is a drawer and one drawer's content ends where the next drawer header starts, here (and in user settings, where it is even worse because the drawer header looks very similar to section headers) that type of clear visual separation is just missing.
When collapsed, the drawer looks like a header for the content coming after it. When expanded, you can't tell where the drawer ends.
Having a header for the next block of content, even if that is not a collapsible drawer, makes it a lot clearer where one thing ends and the next begins.
And don't try and sell me the hr as a visual separation. That thing is styled so faint, it might just as well not be there at all.

**Comment by Cohee1207** [comment]
Even if the header is needed for better visualities, I'd prefer it being something like "World Editor" instead, cause "World Settings" lacks... settings.

**Comment by Cohee1207** [comment]
image
Here's a little idea: make drawer content "drown" in the surrounding background the same way inputs do (border + darker background).

**Comment by LenAnderson** [comment]
That would help with one half of it. Even better if the content wasn't disconnected from the header.

**Comment by Cohee1207** [comment]
Done. What's the second part?

**Comment by LenAnderson** [comment]
Making the collapsed drawer not look like a header for the loose content below it.

**Comment by Wolfsblvt** [comment]
We circling back to my original intention.... ((:

**Review by Cohee1207 — CHANGES_REQUESTED** [review]
I give up on this, my UX vision is greatly impaired.
Just let me know when to merge.

**Comment by Wolfsblvt** [comment]
Let's just postpone this until the other WI reworks.
We need to add more settings anyway with the new global stuff. I'll think about it again then.

**Comment by Cohee1207** [comment]
Feel free to reopen when you feel like it.
